README.md

```diff
@@ -1,6 +1,22 @@
-# Steamship Python Client Library
+<img src="docs/_static/Steamship-symbol-dark.png" alt="Steamship Logo" width="100"/>
 
-Steamship is the fastest way to build, ship, and use full-lifecycle language AI.
+# Steamship Python SDK
+
+[Steamship](https://steamship.com) is the fastest way to build, ship, and use full-lifecycle AI. 🚢 
+
+This library is for:
+- Building and deploying `Packages` and `Plugins` on Steamship
+- Making client calls to a Steamship `Package` from a Python environment
+
+**Are you trying to build a Bot or Agent? Please check out our starter project and guidebook here:**
+
+[Multimodal Agent Starter](https://github.com/steamship-core/multimodal-agent-starter)
+
+[Agent Guidebook](https://www.steamship.com/learn/agent-guidebook)
+
+**Are you trying to host a Langchain agent you've written?  Check out our langchain compatibility library here:**
```

Comment:
we should probably use the LC emojis here, and be consistent about capitalization (as we should expect from their repos).

Reply:
👍